src/en/about/guide.md

```diff
@@ -1,57 +1,105 @@
+
+*The word “fediverse” (federated universe) refers to the network of all Lemmy servers [and other projects](https://blog.joinmastodon.org/2018/06/why-activitypub-is-the-future/), users of which are able to talk to each other seamlessly.*
+
+Like email, wherever you sign up on GMail or Outlook, you know you’ll be able to email everyone you need to so long as you know their address. On Lemmy you'll be able to subscribe to communities on any other server, and can have discussions with users registered elsewhere.
+
+This means there’s no big uncaring company running it all, no shareholders, no central control, none of the targeted advertising we’re all fed up with, just a bunch of people sharing the things they want to share with each other.
+
+## Where do I sign up?
+
+The first thing you have to do is **choose your server**. This is an extra step compared to sites like Reddit, but isn’t as difficult as it may seem.
+
+*Like with email, your identity is hosted by the server you sign up on. So for example, I joined lemmy.ml so to mention me you can type @retiolus@lemmy.ml in your post.*
+
+If what you want to talk about most fits into a clear category (maybe it’s video games or art or queer life or coding or fiction or whatever) then it might be worth making your first server one where that will primarily host that sort of content – it’ll be easier to make connections and find like-minded folk. Some consider your server to be like a neighbourhood or a venue where the general chatter can have a specific focus.
+
+*You have the ability to view all public local posts that have been made by people on your server in the so-called “Local” tab.*
+
+If you aren’t here to stick mainly to one topic, you’ll likely want to pick a server that caters to general interests. Either way, there’s a helpful servers list on [join-lemmy.org](https://join-lemmy.org/instances).
+
+**Don’t panic!** You’ll be able to chatter to people on other servers, no matter which one you choose. Remember, this is like email and you can still email your mum on her ancient Yahoo account from your GMail, for example.
+
+## Knowing your server
+
+Take a moment before registering to check out the rules of your chosen server and make sure they are compatible with how you want to be posting.
+
+Servers usually define their rules in a simple way on the sidebar on the right of the site. From top to bottom, you will usually find the list of "Trending communities" followed by "Subscribed to communities", a brief description of the server, the rules, some stats and the username of the administrators.
+
+Admins are super friendly people who are usually running the server out of their own pocket so it’s good to get to know them like you would a landlord. Many accept donations to cover the running costs and if you’re able to contribute then it’s always appreciated.
```

Comment:
I dont think landlord is a great comparison, your instance admin wont throw you out if you dont pay them :D